notebooks/ICESat-2_Cloud_Access/nsidc_daac_uwg_cloud_access_tutorial.ipynb

```diff
@@ -53,7 +53,7 @@
     "### **Example of end product** \n",
```

Comment:
<p>Suggest "You can use the <code>search_data</code> method to search for granules within a data set by location and time using spatial and temporal filters.  </p><p> </p><p>In this example, you will search for data granules from the ATL06 version 006 cloud-hosted dataset over the Juneau Icefield, AK.  The <code>count = 100</code> keyword requests the first 100 granules. "</p><p> </p><p>Is count required here?  Are we setting count because we want to make sure we get more than the default number of granules or less?  If we include the <code>count</code> keyword it needs to be explained.  Note <code>count</code> is not described in the docstring for <code>search_data</code>.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Have updated text and removed the `count = 100`